addons/website_event/tests/test_event_internals.py

```diff
@@ -19,7 +19,23 @@ def test_menu_create(self):
             'website_menu': True,
             'community_menu': False,
         })
-        self._assert_website_menus(event)
+        self.assertTrue(event.website_menu)
+        self.assertFalse(event.community_menu)
+        self._assert_website_menus(event, ['Introduction', 'Location', 'Register'], menus_out=['Community'])
+
+        event.community_menu = True
+        self._assert_website_menus(event, ['Introduction', 'Location', 'Register', 'Community'])
+
+        event = self.env['event.event'].create({
+            'name': 'TestEvent',
+            'date_begin': fields.Datetime.to_string(datetime.today() + timedelta(days=1)),
+            'date_end': fields.Datetime.to_string(datetime.today() + timedelta(days=15)),
+            'website_menu': False,
+        })
+        self.assertFalse(event.website_menu)
+        self.assertFalse(event.community_menu)
+        self.assertFalse(event.menu_id)
```

Comment:
Could you also call assert_website_menus and set "menu_out" to the whole list?
(And menu_in to an empty list).

Reply:
Indeed

Reply:
Actually it would make no sense, as it searches on menus having event.menu_id as parent, it will automatically find no menus.